README.md

```diff
@@ -127,6 +129,21 @@ OPTIONS
 
 _See code: [src/commands/app/add/web-assets.js](https://github.com/adobe/aio-cli-plugin-app/blob/1.2.3/src/commands/app/add/web-assets.js)_
 
+## `@adobe/aio-cli-plugin-app app:add:ci`
+
+Add CI files (github action workflows under .github folder)
+
+```
+USAGE
+  $ @adobe/aio-cli-plugin-app app:add:ci
+
+OPTIONS
+  -v, --verbose   Verbose output
+  --version       Show version
+```
+
+_See code: [src/commands/app/add/ci.js](https://github.com/adobe/aio-cli-plugin-app/blob/1.2.3/src/commands/app/add/ci.js)_
+
```

Comment:
We should also mention that users will need to configure GitHub secrets as per the workflow files.